### PR TITLE
docs: run only on new tag note on github actions

### PR DESCRIPTION
**Pull request by crazy-max**
refs https://github.com/goreleaser/goreleaser-action/issues/61#issuecomment-569286582

### CODE REVIEW

File: www/content/actions.md [hunk 1]
```diff
@@ -79,3 +81,4 @@ command and a private key without a passphrase.
 [goreleaser-action]: https://github.com/goreleaser/goreleaser-action
 [actions]: https://github.com/features/actions
 [syntax]: https://help.github.com/en/articles/workflow-syntax-for-github-actions#About-yaml-syntax-for-workflows
+[only-new-tag]: https://github.com/goreleaser/goreleaser-action/issues/61#issuecomment-569286582
```
Comment (andreynering) [inline www/content/actions.md]: I think that actually adding the example (see below) with a short description to the README would be better than linking the issue. That's not a strong opinion, though. 🙂 

```yaml
on:
  push:
    tags:
      - '*'
```

Comment (caarlos0) [reply www/content/actions.md]: I agree.. 🤔

Comment (crazy-max) [reply www/content/actions.md]: Done :)

**Comment by caarlos0** [comment]
thanks 🚀

**Pull request merged**